app/assets/javascripts/controllers/tag_groups/controller.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import ColorPick from 'components/tag_groups/ColorPick';
+
+export default () => {
+  const bgColor = $('[name=bg-color]').data('current-color') || "#F17013";
+
+  ReactDOM.render(<ColorPick color ={ bgColor }/>, document.getElementById('tag_group_color'));
+
+  $(document).ready(function(){
```

Comment:
The controllers are already run inside a `$()` (https://github.com/Codeminer42/cm42-central/blob/master/app/assets/javascripts/app.js#L13) (that is an alias to `$(document).ready()`) so there's no need to do that.